### DISCUSSION TITLE
weather in poland

### DISCUSSION THREAD

**Discussion by Krystian-Piwowarski** [chubin/wttr.in#1044] [General]
I have made myself a watch on esp with a display, I have been tweaking the code for a long time but one thing is interesting, we currently have a temperature below zero in Wroclaw but both the web service and my code based on this sees that it is 0 degrees, what is the problem?

from esp
10:18:27.596 -> Raw load: Sunny +0°C 9km/h

from www
Cloudless +0°C 7km/h

**Comment by chubin** [comment]
It is possible that the weather on wttr.in is sometimes slightly off. There are basically two reasons for this:

1. Accuracy of the data source
2. Caching issues

We are working on both of these, but it will take some time to fix them.

As a workaround, I propose you add something changing the URL (like xxxx=$RANDOM) to bypass at least one of the caching layers.
